plugin/pkg/shoot/validator/admission.go

```diff
@@ -843,6 +843,41 @@ func (c *validationContext) validateAdmissionPlugins(a admission.Attributes, sec
 	return allErrs
 }
 
+// For backwards-compatibility, we want to validate the scaleDownUnneededTime and scaleDownDelayAfterAdd only for newly created Shoot clusters.
+// Performing the validation for all Shoots would prevent already existing Shoots with the wrong spec to be updated/deleted.
+// Validation implies that ScaleDownDelayAfterAdd and ScaleDownUnneededTime shouldn't be less than a minute at the same time to avoid scaling cycles.
+func (c *validationContext) validateScaleDownConfiguration(a admission.Attributes) field.ErrorList {
```

Comment:
`scaleDownConfigution` is a common term that can be used in the context of cluster-autoscaler or vertical-pod-autoscaler. You are validating the cluster-autoscaler fields:

```suggestion
func (c *validationContext) validateClusterAutoscaler(a admission.Attributes) field.ErrorList {
```